source/kraken/tests/fill_disruption_from_chaos_tests.cpp

```diff
@@ -55,7 +55,7 @@ namespace ntest = navitia::test;
 namespace bt = boost::posix_time;
 namespace ba = boost::algorithm;
 
-
+/*
```

Comment:
It's used on the test that are commented after, about desactivating a stop point.